src/source/Crypto/Dtls_mbedtls.c

```diff
@@ -3,8 +3,9 @@
 
 /**  https://tools.ietf.org/html/rfc5764#section-4.1.2 */
 mbedtls_ssl_srtp_profile DTLS_SRTP_SUPPORTED_PROFILES[] = {
-    MBEDTLS_SRTP_AES128_CM_HMAC_SHA1_80,
-    MBEDTLS_SRTP_AES128_CM_HMAC_SHA1_32,
+    MBEDTLS_TLS_SRTP_AES128_CM_HMAC_SHA1_80,
+    MBEDTLS_TLS_SRTP_AES128_CM_HMAC_SHA1_32,
+    MBEDTLS_TLS_SRTP_UNSET,
```

Comment:
for my info - what exactly the unset profile means here?

Reply:
oh, I see, they must have changed the API to have a terminator value?

Reply:
Yes, they updated the API, it now uses a terminator value